specification/deviceprovisioningservices/Deviceprovisioningservices.Management/models.tsp

```diff
@@ -0,0 +1,872 @@
+  S1: "S1",
+}
+
+/**
+ * Type of managed service identity (where both SystemAssigned and UserAssigned types are allowed).
+ */
+union ManagedServiceIdentityType {
+  string,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  None: "None",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  SystemAssigned: "SystemAssigned",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  UserAssigned: "UserAssigned",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  `SystemAssigned,UserAssigned`: "SystemAssigned,UserAssigned",
+}
+
+/**
+ * specifies the reason a name is unavailable
+ */
+union NameUnavailabilityReason {
+  string,
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  Invalid: "Invalid",
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  AlreadyExists: "AlreadyExists",
+}
+
+/**
+ * The desired action for requests captured by this rule.
+ */
+#suppress "@azure-tools/typespec-azure-core/no-enum" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+enum IpFilterActionType {
+  Accept,
+  Reject,
+}
+
+/**
+ * Target for requests captured by this rule.
+ */
+#suppress "@azure-tools/typespec-azure-core/no-enum" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+enum IpFilterTargetType {
+  all,
+  serviceApi,
+  deviceApi,
+}
+
+/**
+ * Error details.
+ */
+@error
+model ErrorDetails {
+  /**
+   * The error code.
+   */
+  @visibility(Lifecycle.Read)
+  code?: int32;
+
+  /**
+   * The HTTP status code.
+   */
+  @visibility(Lifecycle.Read)
+  httpStatusCode?: string;
+
+  /**
+   * The error message.
+   */
+  @visibility(Lifecycle.Read)
+  message?: string;
+
+  /**
+   * The error details.
+   */
+  @visibility(Lifecycle.Read)
+  details?: string;
+}
+
+/**
+ * The description of an X509 CA Certificate.
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+model CertificateProperties {
+  /**
+   * The certificate's subject name.
+   */
+  @visibility(Lifecycle.Read)
+  subject?: string;
+
+  /**
+   * The certificate's expiration date and time.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-format" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @visibility(Lifecycle.Read)
+  @format("date-time-rfc1123")
```

Comment:
`format` has been deprecated.  please resolve before merging.
<img width="513" height="256" alt="image" src="https://github.com/user-attachments/assets/0004edff-36dc-40b5-9c65-7eb3b10b33dd" />
This change is not desired.